src/black/__init__.py

```diff
@@ -69,7 +69,8 @@
 DEFAULT_INCLUDES = r"\.pyi?$"
 CACHE_DIR = Path(user_cache_dir("black", version=__version__))
 
-STRING_PREFIX_CHARS: Final = "furbFURB"  # All possible string prefix characters.
+# All possible string prefix characters.
+STRING_PREFIX_CHARS: Final = "furbFURB"
```

Comment:
NIT: IMO this is unnecessary and just ruins blame information.

Reply:
was a result of autoformatter. I see the point. will revert.

Reply:
Fixed (also all the rest)